west.yml

```diff
@@ -143,7 +143,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 352ea3d4a6917df35392dde942e6a36a6c3f802a
+      revision: pull/1170/head
```

Comment:
please update to sha after https://github.com/nrfconnect/sdk-nrfxlib/pull/1170 is merged